app/src/main/java/to/bitkit/ui/ContentView.kt

```diff
@@ -337,99 +352,142 @@ fun ContentView(
+                                            onContinue = {
+                                                appViewModel.dismissTimedSheet(skipQueue = true)
+                                                navController.navigate(Routes.QuickPaySettings)
+                                            },
+                                        )
+                                    }
+
+                                    TimedSheetType.HIGH_BALANCE -> {
+                                        HighBalanceWarningSheet(
+                                            understoodClick = { appViewModel.dismissTimedSheet() },
+                                            learnMoreClick = {
+                                                val intent =
+                                                    Intent(Intent.ACTION_VIEW, Env.STORING_BITCOINS_URL.toUri())
+                                                context.startActivity(intent)
+                                                appViewModel.dismissTimedSheet(skipQueue = true)
+                                            }
+                                        )
+                                    }
                                 }
                             }
                         }
                     }
+                ) {
+                    RootNavHost(
+                        navController = navController,
+                        drawerState = drawerState,
+                        walletViewModel = walletViewModel,
+                        appViewModel = appViewModel,
+                        activityListViewModel = activityListViewModel,
+                        settingsViewModel = settingsViewModel,
+                        currencyViewModel = currencyViewModel,
+                        transferViewModel = transferViewModel,
+                    )
                 }
-            ) {
-                RootNavHost(
-                    navController = navController,
-                    walletViewModel = walletViewModel,
-                    appViewModel = appViewModel,
-                    activityListViewModel = activityListViewModel,
-                    settingsViewModel = settingsViewModel,
-                    currencyViewModel = currencyViewModel,
-                    transferViewModel = transferViewModel,
+
+                val navBackStackEntry by navController.currentBackStackEntryAsState()
+
+                val currentRoute = navBackStackEntry?.destination?.route
+
+                val showTabBar = currentRoute in listOf(
+                    Routes.Home::class.qualifiedName,
+                    Routes.AllActivity::class.qualifiedName,
+                )
+
+                AnimatedVisibility(
+                    visible = showTabBar && currentSheet == null,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .align(Alignment.BottomCenter)
+                ) {
```

Comment:
Would've been nicer to move it below the visible area of the screen, like iOS does :)